src/poliastro/plotting.py

```diff
@@ -394,6 +394,226 @@ def savefig(self, filename, **layout_kwargs):
         )
 
 
+def _generate_circle(radius, center, num=500):
+    u1 = np.linspace(0, 2 * np.pi, num)
+    x_center, y_center, z_center = center
+
+    xx = x_center + radius * np.cos(u1)
+    yy = y_center + radius * np.sin(u1)
+
+    return xx, yy
+
+
+class OrbitPlotter2D:
+    """OrbitPlotter2D class.
+
+    """
+
```

Comment:
Extra whitespace?

Reply:
Oh! Will remove that!

Reply:
I followed the way ``OrbitPlotter3D`` is written

Reply:
https://github.com/shreyasbapat/poliastro/blob/5da5388dd111df0a9701071530e1a2c5920ca21c/src/poliastro/plotting.py#L236